demo/src/pages/Search.js

```diff
@@ -56,6 +56,7 @@ const Search = () => {
     };
 
     const handleRouteChange = (searchObj, hash, refreshId) => {
+        console.log(searchObj, hash, refreshId,"searchObj, hash, refreshId")
```

Comment:
Let's remove this console log

Reply:
its in the demo. so wont effect the sdk

Reply:
Since it is not useful, it is generally not a good practice to leave unnecessary code in the final branch.

Reply:
removed